### ISSUE TITLE
Moving a folder always results in an error

### ISSUE THREAD

**Issue by ChristopherHale** [krathjen/studiolibrary#384]
When moving a folder in studio library it consistently throws this error.

```
RenamePathError: Cannot save over an existing path: "X:/Dauntless/Tools/Animation/StudioLibrary/Monsters" // 
# Traceback (most recent call last):
#   File "C:/perforce/Dauntless/Archon/Tools/MayaTools\studiolibrary\src\studiolibrary\libraryitem.py", line 791, in showMoveDialog
#     self.move(dst)
#   File "C:/perforce/Dauntless/Archon/Tools/MayaTools\studiolibrary\src\studiolibrary\libraryitem.py", line 715, in move
#     self.rename(dst)
#   File "C:/perforce/Dauntless/Archon/Tools/MayaTools\studiolibrary\src\studiolibrary\libraryitem.py", line 732, in rename
#     dst = studiolibrary.renamePath(src, dst)
#   File "C:/perforce/Dauntless/Archon/Tools/MayaTools\studiolibrary\src\studiolibrary\utils.py", line 758, in renamePath
#     raise RenamePathError(msg.format(dst))
# studiolibrary.utils.RenamePathError: Cannot save over an existing path: "X:/Dauntless/Tools/Animation/StudioLibrary/Monsters"
```

**Comment by ChristopherHale** [comment]
I tried updating the Move function to have similar behavior to the MoveToTrash function. It seems to work but I'm not familiar with your code.

libraryitem.py Line 708
```
    def move(self, dst):
        """
        Move the current item to the given destination.

        :type dst: str
        :rtype: None
        """
        #self.rename(dst)
        path = self.path()
        library = self.library()
        item = studiolibrary.LibraryItem(path, library=library)
        self.libraryWindow().moveItems([self], dst)
```

**Comment by krathjen** [comment]
Thanks for reporting the issue. This should be fixed in version 2.9.11
